ui/packages/catalog-ui-search/src/main/webapp/js/widgets/cesium.line.js

```diff
@@ -54,6 +54,12 @@ class LineRenderView extends GeometryRenderView {
     if (!linePoints) {
       return
     }
+
+    linePoints.map(point => {
```

Comment:
```suggestion
    linePoints.forEach(point => {
```

Reply:
This is necessary, since map does not modify the array.